# Plant Breeder & Data analyst

#### Summary: 
Ph.D. in Plant Breeding and Genetics with 6 years of research experience in genetics, genomics, and data analysis of crop plants. Skilled in experimental design, statistical analysis, and development of R packages for plant breeding applications. Interested in a research or industry role involving genetics, plant breeding, bioinformatics, or Artificial Intelligence for plant breeding. 

#### Soft Technical Skills: R, Shiny, Python, Google colab

## Education
- **Ph.D.**, Genetics and Plant Breeding | [TamilNadu Agricultural University](https://tnau.ac.in/) (_Thesis submitted on 31st October 2023_)								       		
- **M.Sc.**, Genetics and Plant Breeding	| [TamilNadu Agricultural University](https://tnau.ac.in/) (_December 2020_)	 			        		
- **B.Sc.**, Agriculture | [College of Agricultural Technology](https://cattheni.edu.in/) (_July 2017_)

## Research Experience
**Ph.D. reseach @[TNAU](https://tnau.ac.in/) (_2021 - 2023_)**
- Developed and applied advanced tissue culture techniques like embryo culture to rapidly <ins>generate somaclonal variants of rice</ins> with improved <ins>salinity stress tolerance</ins>.
- Identified and validated <ins>novel genomic regions (QTLs) governing salinity tolerance</ins> through molecular mapping in rice F<sub>2</sub> populations along with thorough phenotypic and biochemical characterization.
- Employed <ins>multi-omics approaches</ins> including GC-MS metabolomics to uncover key genes and metabolites linked to superior osmotic/ionic adjustment under salt stress.

**Research scholar @ [ICRISAT](https://www.icrisat.org/) (_2018 - 2020_)**
- Single plant phenotying of diverse germplasm accessions (Sorghum, Peral millet, Pigeon pea) to examine intra and inter accession genetic diversity. 
- Single plant genotyping of accessions using [DArTSeq](https://www.diversityarrays.com/) based SNPs.
- Genomic data analysis and development of statistical models for estimating optimum sample sizes.

## Work experience 
**Data Science Consultant @ [fiverr](https://www.fiverr.com/s/KzDd5k) (_Decembe 2020 - Present_)**
- **5-star rated** expert in R data analysis and visualization with over six years of experience specializing in genomic data analysis and bioinformatics.
- Proficient in handling big data and performing complex modelling using popular R libraries and packages such as tidyr, data.table, dplyr, plyr, tenserflow, ggplot, ggdendro, ggtree, ggheatmap, and circos. 
- Received positive feedback from clients for knowledge, professionalism, and mastery of R programming. Demonstrated ability to deliver high-quality work as evidenced by the portfolio available on the [profile](https://www.fiverr.com/s/KzDd5k). 

## Programming and data analysis skills
- Proficient in full stack development of **R packages** using modular coding practices
- Created production-grade **Shiny web applications** for interactive data analysis and visualization and; expertise in dependency management tools like Golem for scalable deployment
- **Machine learning model** building for image classification/segmentation tasks; trained CNNs and other deep learning architectures in R, Python (PyCharm) 
- **Multivariate data analysis** of large-scale omics datasets including genomics, phenomics and metabolomics using cutting-edge bioinformatics tools
- **Advanced visualization** for multi-dimensional biological data through Circos, ggtree, ggtreeextra, Cytoscape and other platforms

**Additional Skills:**
  
- Git/GitHub for version control and collaborative coding 
- High performance computing on clusters for scalable data analysis
- Bioconductor for analyzing genomic/transcriptomic experiments
- Workflow automation to enhance reproducibility and to faster workflow. 

## R Packages developed
### PBPheno
[CRAN](https://www.mdpi.com/1424-8220/22/8/3048)

Developed Using R....

### PBGeno
[CRAN](https://www.mdpi.com/1424-8220/22/11/4240)

Used R....

### PBPerfect
**PBPerfect** [(vist page)](https://allanbiotools.shinyapps.io/pbperfect/) is a interactive web tool enabling reproducible multivariate analysis with visualization of phenotypic and genotypic data. It features basic statistics, experimental designs, SSR workflows, multivariate analysis, mating designs, and dynamic graphics with outputs exported as publication-standard tables and graphics requiring no further formatting.

## Intenship and Conferences
- 
## Talks & Lectures
- Seminars and Lectures

- [Medium articles](https://www.youtube.com/channel/UCa9gErQ9AE5jT2DZLjXBIdA)

## Publications
1. Talebi S., Lary D.J., Wijeratne L. OH., and Lary, T. Modeling Autonomic Pupillary Responses from External Stimuli Using Machine Learning (2019). DOI: 10.26717/BJSTR.2019.20.003446
2. Wijeratne, L.O.; Kiv, D.R.; Aker, A.R.; Talebi, S.; Lary, D.J. Using Machine Learning for the Calibration of Airborne Particulate Sensors. Sensors 2020, 20, 99.
3. Lary, D.J.; Schaefer, D.; Waczak, J.; Aker, A.; Barbosa, A.; Wijeratne, L.O.H.; Talebi, S.; Fernando, B.; Sadler, J.; Lary, T.; Lary, M.D. Autonomous Learning of New Environments with a Robotic Team Employing Hyper-Spectral Remote Sensing, Comprehensive In-Situ Sensing and Machine Learning. Sensors 2021, 21, 2240. https://doi.org/10.3390/s21062240
4. Zhang, Y.; Wijeratne, L.O.H.; Talebi, S.; Lary, D.J. Machine Learning for Light Sensor Calibration. Sensors 2021, 21, 6259. https://doi.org/10.3390/s21186259
5. Talebi, S.; Waczak, J.; Fernando, B.; Sridhar, A.; Lary, D.J. Data-Driven EEG Band Discovery with Decision Trees. Preprints 2022, 2022030145 (doi: 10.20944/preprints202203.0145.v1).
6. Fernando, B.A.; Sridhar, A.; Talebi, S.; Waczak, J.; Lary, D.J. Unsupervised Blink Detection Using Eye Aspect Ratio Values. Preprints 2022, 2022030200 (doi: 10.20944/preprints202203.0200.v1).
7. Talebi, S. et al. Decoding Physical and Cognitive Impacts of PM Concentrations at Ultra-fine Scales, 29 March 2022, PREPRINT (Version 1) available at Research Square [https://doi.org/10.21203/rs.3.rs-1499191/v1]
8. Lary, D.J. et al. (2022). Machine Learning, Big Data, and Spatial Tools: A Combination to Reveal Complex Facts That Impact Environmental Health. In: Faruque, F.S. (eds) Geospatial Technology for Human Well-Being and Health. Springer, Cham. https://doi.org/10.1007/978-3-030-71377-5_12
9. Wijerante, L.O.H. et al. (2022). Advancement in Airborne Particulate Estimation Using Machine Learning. In: Faruque, F.S. (eds) Geospatial Technology for Human Well-Being and Health. Springer, Cham. https://doi.org/10.1007/978-3-030-71377-5_13

## Contact

| **Name**   | Allan. V | 
| **E-mail**   | [albertoogy@gmail.com](mailto:albertoogy@gmail.com) | 
| **LinkedIn**   | [Visit page](https://www.linkedin.com/in/allan-victor-6919b4152/) | 
